scope3_methodology/api/api.py

```diff
@@ -299,5 +314,25 @@ def parse_corporate_public_yaml_file(identifier: str):
         ) from exc
 
 
+@app.get("/defaults/end_user_device")
+def get_all_end_user_device_defaults():
+    """
+    Returns all end user device defaults
+    """
+    response = []
+    for device, device_defaults in end_user_device_defaults.items():
+        for channel, channel_defaults in property_defaults.items():
+            if channel_defaults.quality_impressions_per_duration_s:
```

Comment:
how/where do we ensure each channel default has this defined?

Reply:
We don't currently have any testing on merge for templates. There is an issue to introduce checks. That being said, the point of this check is to ensure we do only calculate emissions for those channels with this set. So anything without full defaults will not be exposed.